src/amo/components/Footer/index.js

```diff
@@ -143,6 +136,81 @@ export class FooterBase extends React.Component {
                 </a>
               </li>
             </ul>
+          </section>
+
+          <section className="Footer-product-links">
+            <h4 className="Footer-links-header">Product</h4>
```

Comment:
This should plural - `Products` and be translated.

Reply:
@diox Done!